src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1421,7 +1428,12 @@ def test_go_loop_swap_errors():
                          "test_files", "dynamo0p3",
                          "1.0.1_single_named_invoke.f90"),
                     api="dynamo0.3")
+    # TODO: that still doesn't work, we have inconsistent values
```

Comment:
Is this still true? If so it sounds like something that needs addressing...

Reply:
Done.